service/provider/src/main/java/gov/va/vro/service/provider/camel/MasIntegrationRoutes.java

```diff
@@ -121,19 +129,33 @@ public void process(Exchange exchange) {
                     (HealthDataAssessment) exchange.getProperty("evidence");
                 if (evidence.getErrorMessage() != null) {
                   log.error("Health Assessment Failed");
+                } else {
+                  exchange.setProperty(
+                      "sufficientForFastTracking", evidence.isSufficientForFastTracking());
+                  log.info(
+                      " MAS Processing >> Sufficient Evidence >>> "
+                          + evidence.isSufficientForFastTracking());
+                  var masTransferObject =
+                      new MasTransferObject(claimPayload, assessment.getEvidence());
```

Comment:
I wonder if we are supposed to pass the evidence from the assessment here or the evidence returned from AbdEvidenceWithSummary? I am not familiar with the exact requiement.